CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### **Added**
 
 ### **Changed**
+- enforce ssl on pipeline bucket in sagemaker templates module
```

Comment:
Did you check all other buckets that we create?

Reply:
This is what was picked up by a scanning tool but checking if there's any other ones that it might have missed

Reply:
Didn't realize these were found by a tool! I checked a bunch and didn't see any issues.

Reply:
Same, I just manually ran through all buckets in code and did not find any misses so looks like we're good